examples/browser/README.md

```diff
@@ -0,0 +1,29 @@
+## 0x Mesh Browser Example
+
+This directory contains an example of how to run Mesh in the browser.
+
+### Running the Example
+
+To run the example, first change into the __examples/browser__ directory and
+then run:
+
+```
+yarn install --force && yarn build
```

Comment:
Why `--force`?

Reply:
`--force` is required because we use a local directory for `@0x/mesh-browser` in the __package.json__ file for the example. This makes it so that you are always using the version of Mesh that was intended for the example code.